dashboard/test/factories.rb

```diff
@@ -349,10 +349,12 @@
     plc_user_course_enrollment nil
     plc_course_unit nil
     status Plc::EnrollmentUnitAssignment::START_BLOCKED
+    user nil
```

Comment:
Since this is `null: false` in the schema, maybe have the default factory create a user? Otherwise it's unusable without supplying one. E.g. `user {create :user}`